bindings/mnt/pyfiction/include/pyfiction/algorithms/physical_design/design_sidb_gates.hpp

```diff
@@ -19,7 +19,7 @@ namespace detail
 {
 
 template <typename Lyt>
-void design_sidb_gates(pybind11::module& m)
+void design_sidb_gates_impl(pybind11::module& m)
```

Comment:
This is a naming inconsistency in how we set up these files usually, or?

Reply:
you are right! I will revert the changes. Thanks!